docs/oathkeeper/index.md

```diff
@@ -3,12 +3,15 @@ id: index
 title: Introduction
 ---
 
-ORY Oathkeeper authorizes incoming HTTP requests, either by being deployed as a
-reverse proxy in front of your upstream server (API, web server, ...) or as a
-Access Control Decision API alongside your API Gateway (Kong, Nginx, Envoy, AWS
-API Gateway, ...).
-
-The implemented problem domain and scope is called Zero-Trust Network
+ORY Oathkeeper authorizes incoming HTTP requests.
+It can be the Policy Enforcement Point in your cloud architecture, 
+i.e. a reverse proxy in front of your upstream API or web server 
+that forwards authorized requests and rejects unauthorized ones.
+If you want to use another API Gateway (Kong, Nginx, Envoy, AWS API Gateway, ...),
+Oathkeeper can also plug into that and act as its Policy Decision Point.
+
+The implemented problem domain and scope is part of [attribute-based access control](https://en.wikipedia.org/wiki/Attribute-based_access_control), 
```

Comment:
Oathkeeper is agnostic to access control mechanisms (ABAC, RBAC, ACL, ...) but instead uses the concept of enforcing policies/permissions which can be implemented in a variety of ways. Currently supported are "allow all requests", "deny all requests", and ORY Keto as a Policy Decision Point.

Reply:
Okay, I have reverted this change. Thanks for the background!